tests/test_kinematics.py

```diff
@@ -118,21 +128,24 @@ def test_urdf_serial():
     th_batch = torch.rand(N, len(chain.get_joint_parameter_names()), dtype=dtype, device=d)
     chain = chain.to(dtype=dtype, device=d)
 
-    import time
-    start = time.time()
-    tg_batch = chain.forward_kinematics(th_batch)
-    m = tg_batch.get_matrix()
-    elapsed = time.time() - start
-    print("elapsed {}s for N={} when parallel".format(elapsed, N))
+    number = 10
 
-    start = time.time()
-    elapsed = 0
-    for i in range(N):
```

Comment:
Line ordering is a bit awkward - how about
```python
for i in range(N):
    start = time.time()
    tg = chain.forward_kinematics(th_batch[i])
    mm = tg.get_matrix().view(4, 4)
    elapsed += time.time() - start
    assert torch.allclose(mm, m[i])
```
Note that `tg.get_matrix()` could be expensive since Transform3d is lazy and doesn't do matrix multiplications / actual transforms until when you need the matrix

Reply:
I think your comment is referring to the old code I deleted? Also I checked and get_matrix isn't affecting performance in this case.